Taskfile.yml

```diff
@@ -0,0 +1,144 @@
+version: "3"
+
+vars:
+  # Variables for Docker image
+  REGISTRY: ghcr.io/openmcp-project
+  IMAGE_NAME: mcp-ui-frontend
+  IMAGE_TAG: '{{.TAG | default "latest"}}'
+  DOCKER_IMAGE: "{{.REGISTRY}}/{{.IMAGE_NAME}}:{{.IMAGE_TAG}}"
+
+  # Variables for local testing
+  LOCAL_REGISTRY: localhost:5000
+  LOCAL_IMAGE_NAME: mcp-ui-frontend
+  LOCAL_DOCKER_IMAGE: "{{.LOCAL_REGISTRY}}/{{.LOCAL_IMAGE_NAME}}:{{.IMAGE_TAG}}"
+
+  # Variables for OCM component
+  OCM_COMPONENT_NAME: github.com/openmcp-project/ui
+  OCM_COMPONENT_VERSION: '{{.OCM_COMPONENT_VERSION | default .IMAGE_TAG}}'
+  OCM_PROVIDER: openmcp-project
+  OCM_TARGET_REPO: '{{.OCM_TARGET_REPO | default (print .REGISTRY "/components") }}'
+  OCM_OUTPUT_DIR: "{{.ROOT_DIR}}/.ctf"
+  OCM_DESCRIPTOR: "{{.ROOT_DIR}}/ocm/component-descriptor.yaml"
+  OCM_DESCRIPTOR_LOCAL: "{{.ROOT_DIR}}/ocm/component-descriptor-local.yaml"
+
+tasks:
+  default:
+    desc: Show available tasks
+    cmds:
+      - task --list
+
+  build:ocm:
+    desc: "Builds the ocm component. Usage: task build:ocm -- <version>"
+    cmds:
+      - rm -rf {{.OCM_OUTPUT_DIR}}
+      - |
+        ocm add components --create --force --copy-resources \
+          --complete \
+          --lookup {{.OCM_TARGET_REPO}} \
+          --file {{.OCM_OUTPUT_DIR}} \
+          {{.OCM_DESCRIPTOR}} -- \
+          VERSION={{.OCM_COMPONENT_VERSION}} \
+          COMPONENT_NAME={{.OCM_COMPONENT_NAME}} \
+          PROVIDER={{.OCM_PROVIDER}}
+    vars:
+      OCM_COMPONENT_VERSION: '{{.CLI_ARGS}}'
+
+  publish:ocm:
+    desc: "Publishes the ocm component to the registry."
+    cmds:
+      - |
+        ocm transfer ctf \
+          --copy-resources \
+          --enforce --overwrite \
```

Comment:
```suggestion
```

No need for copy-resources, the image is already in ghcr. This would copy it again.
Would prefer not to use --enforce and --overwrite to not overwrite a component for now.